h/security/predicates.py

```diff
@@ -137,11 +138,6 @@ def group_joinable_by_authority(_identity, context):
     return context.group.joinable_by == JoinableBy.authority
 
 
-@requires(authenticated_user, group_found)
-def group_created_by_user(identity, context):
-    return context.group.creator and context.group.creator.id == identity.user.id
```

Comment:
No longer needed. Being the creator of a group no longer grants you any permissions.